src/RazorpayClient.cs

```diff
@@ -1,18 +1,82 @@
+using System.Collections.Generic;
+
 namespace Razorpay.Api
 {
     public class RazorpayClient
     {
-        public const string Version = "1.0.0";
-        public const string BaseUrl = "https://api.razorpay.com/v1/";
-        public static string Key = null;
-        public static string Secret = null;
+        protected const string version = "1.2.0";
+        protected const string baseUrl = "https://api.razorpay.com/v1/";
+        protected static string key = null;
+        protected static string secret = null; 
+        protected static List<Dictionary<string, string>> appsDetails = new List<Dictionary<string, string>>();
         private Payment payment = null;
         private Order order = null;
+        private Customer customer = null;
+        private Invoice invoice = null;
+        private Token token = null;
+        private Card card = null;

```

Comment:
@amitmahbubani But you get a transfer from the payment entity, and a reversal from the transfer entity. So why define it here as well?